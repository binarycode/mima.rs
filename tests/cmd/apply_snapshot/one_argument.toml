args = 'apply-snapshot one'
status.code = 2
fs.sandbox = true
