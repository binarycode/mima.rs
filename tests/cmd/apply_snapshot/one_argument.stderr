error: the following required arguments were not provided:
  <SNAPSHOT_ID>

Usage: mima apply-snapshot <GUEST_ID> <SNAPSHOT_ID>

For more information, try '--help'.
